modules/apps/digital-signature/digital-signature-test/src/testFunctional/tests/DigitalSignatureSiteSettings.testcase

```diff
@@ -37,159 +39,65 @@ definition {
 	@description = "This is a test for LPS-131427. Verify if Digital Signature can be enable in Site Settings, using the Always Override Strategy"
 	@priority = 5
 	test CanEnableDigitalSignatureBySiteSettings {
-		var siteStrategy = "Always Override";
-
-		Select(
-			locator1 = "ClickToChatConfiguration#SITE_SETTINGS_STRATEGY",
-			value1 = ${siteStrategy});
-
-		SystemSettings.saveConfiguration();
-
-		Navigator.openURL();
-
-		ProductMenu.gotoPortlet(
-			category = "Configuration",
-			portlet = "Site Settings");
-
-		Click(
-			key_itemName = "Digital Signature",
-			locator1 = "ListGroupItem#ITEM_TEXT");
-
-		var digitalSignatureApiUsername = PropsUtil.get("digital.signature.api.username");
-		var digitalSignatureApiAccountId = PropsUtil.get("digital.signature.api.accountId");
-		var digitalSignatureAccountBaseUri = PropsUtil.get("digital.signature.account.base.uri");
-		var digitalSignatureIntegrationKey = PropsUtil.get("digital.signature.integration.key");
-		var digitalSignatureRsaPrivateKey = PropsUtil.get("digital.signature.rsa.private.key");
-
-		Type(
-			locator1 = "DigitalSignatureConfiguration#DIGITAL_SIGNATURE_USER_ID",
-			value1 = ${digitalSignatureApiUsername});
+		task ("Given the digital signature using the Always Override strategy") {
+			DigitalSignature.siteSettingsStrategy(siteStrategy = "Always Override");
 
-		Type(
-			locator1 = "DigitalSignatureConfiguration#DIGITAL_SIGNATURE_ACCOUNT_ID",
-			value1 = ${digitalSignatureApiAccountId});
-
-		Type(
-			locator1 = "DigitalSignatureConfiguration#DIGITAL_SIGNATURE_BASE_URI",
-			value1 = ${digitalSignatureAccountBaseUri});
+			SystemSettings.saveConfiguration();
+		}
 
-		Type(
-			locator1 = "DigitalSignatureConfiguration#DIGITAL_SIGNATURE_INTEGRATION_KEY",
-			value1 = ${digitalSignatureIntegrationKey});
+		task ("When the credentials are placed and the digital signature is created") {
```

Comment:
Please, change this task to:` When the user set the credentials  by site settings`